### PR TITLE
feat: apply sorting by amount by sales in admin.sales page

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
apply sorting by amount by sales in admin.sales page

Required for https://github.com/fossasia/open-event-frontend/pull/7642

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -62,3 +62,4 @@ celerybeat-schedule*
 .pytype
 .tool-versions
 *.mo
+.python-version
```
Comment (maze-runnar) [inline .gitignore:65]: why?

**Comment by maze-runnar** [comment]
resolve conflict. 
changes are LGTM!!

File: .gitignore (hunk 1 above)
Comment (MeghalBisht) [reply .gitignore:65]:
```suggestion
```

**Review by maze-runnar — APPROVED** [review]
Resolve lgtm issue.

File: app/api/admin_sales/events.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 from app.api.admin_sales.utils import event_type, summary
```
Comment (MeghalBisht) [inline app/api/admin_sales/events.py:5]:
```suggestion
from app.api.admin_sales.utils import summary
```

**Pull request merged**